newton/examples/__init__.py

```
@@ -398,34 +398,29 @@ def init(parser=None):
     """Create a collision pipeline, optionally using --broad-phase-mode from args.
 
     Args:
         model: The Newton model to create the pipeline for.
         args: Parsed arguments from create_parser() (optional).
+        broad_phase: Override broad phase ("nxn", "sap", "explicit"). Default from args or "explicit".
         **kwargs: Additional keyword arguments passed to CollisionPipeline.
 
     Returns:
         CollisionPipeline instance.
     """
     import newton  # noqa: PLC0415
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)